releases/values/teleport-ent/teleport-ent-roles.yaml.gotmpl

```diff
@@ -53,6 +53,39 @@ bootstrap:
               verbs: [delete]
             - resources: [trusted_cluster]
               verbs: [create, update]
+    power-user.yaml: |
+      kind: "role"
+      version: "v3"
+      metadata:
+        name: "power-user"
+      spec:
+        options:
+          # max_session_ttl defines the TTL (time to live) of SSH certificates
+          # issued to the users with this role.
+          max_session_ttl: "{{ env "TELEPORT_SESSION_TTL" | default "4h" }}"
+          port_forwarding: false
+        allow:
+          # Deny SSH access
```

Comment:
Allow SSH access?

Reply:
or does it mean deny if logins and node_labels are empty?

Reply:
The intent is to Deny SSH access. It is really accomplished by the rule under `deny`, but this is an effort to prohibit it via `allow`.